README.md

```diff
@@ -0,0 +1,78 @@
+PHP client for integrating Prospect into your projects.
+=======================================================
+
+Example usage
+-------------
+
+### Create a new report, poll for it to be completed, then display the report
+
+```php
+use Silktide\ProspectClient\Api\ReportApi;
+use Silktide\ProspectClient\Api\ReportInProgressException;
+
+// Keys can be loaded from the environment or passed to your script. 
+// The actual API key needs to be created at https://app.prospect.silktide.com/en_GB/admin/settings#/api
+define("PROSPECT_API_KEY", "0123456789abcdef");
+
+$reportApi = new ReportApi(PROSPECT_API_KEY);
```

Comment:
Talking point: This method of initialisation feels a bit weird to me. If we created a number of separate APIs (say, ReportApi, ListApi, StatisticsApi for example), would we be expecting to do an initialisation for each of them with the API key? 

The only company that does that that comes to mind is AWS and largely as each service has custom global parameters to that service (region, version et al)

From my point of view, I think I'd expect something more akin to a `SilktideClient` class with either `->report()->fetch(...` sort of vibes or just `reportFetch(...`

Thoughts?

Reply:
*ProspectClient, not SilktideClient even